### PR TITLE
milestone 5

### PR DESCRIPTION
**Pull request by milosvukadinovic**

### CODE REVIEW

**Comment by milosvukadinovic** [comment]
My linter had some problems so hopefully everything is good now.

File: src/Components/App.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+class App extends React.Component {
+  state={
+    total: null,
+    next: null,
+    operation: null,
+    error: null,
+  }
+
+handleClick = (buttonName) => {
+  this.setState(calculate(this.state, buttonName));
```
Comment (BalintMendli) [inline src/Components/App.js]: [Optional] `setState` is asynchronous so when we are using previous state to calculate the next state it's better to use the setState with a callback function to ensure we use the correct state:
`this.setState(prevState => calculate(prevState, buttonName)`

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @milosvukadinovic ,

Great job. 👏 

We are almost done. We just have to fix two issues:

- There are two `+` buttons on the calculator and no `/`. Please fix this.

- We don't have error handling implemented yet. Dividing by 0 blows up the app.

Please fix these things and submit another review request.

**Comment by milosvukadinovic** [comment]
@zenott 
Greetings Balint and thank you for reviewing my code.
I am having trouble with deploying to Heroku and cannot seem to figure out the problem.
Could you take a look?
I'm having the following error during deployment:

-----> Installing dependencies
       Installing node modules (package.json + package-lock)
       npm ERR! code EINTEGRITY
       npm ERR! sha512-z3U7eMY6r/3f3/JB9mTsLjyxrv0Yb1zb8PCWCLpguxfCzBIZUwy23R1t/XKewP+8mEN2Ck8Dtr4q20z6ce6SoA== integrity checksum failed when using sha512: wanted sha512-z3U7eMY6r/3f3/JB9mTsLjyxrv0Yb1zb8PCWCLpguxfCzBIZUwy23R1t/XKewP+8mEN2Ck8Dtr4q20z6ce6SoA== but got sha512-UdsurWPtgiPgpJ06ryUnuaSXC2s0WoSZnQmEpbAH65XZSdwowgN5MvyP7e88nW07FYXv72erVtpBkxyDVKhH1Q==. (9268 bytes)
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.KdoTS/_logs/2019-09-24T10_07_18_973Z-debug.log
-----> Build failed

**Comment by milosvukadinovic** [comment]
It has something to do with babel Eslint integrity I am looking into it.

**Comment by milosvukadinovic** [comment]
Issue with integrity fixed

**Review by BalintMendli — APPROVED** [review]
Hi @milosvukadinovic ,

Congrats, the calculator works great now. Great that you've solved the deployment issue also. _(Please contact me on Slack next time if you have questions, I often miss Github comments in between the myriad of Github notifications)_

This is a
### 👍 

You can go ahead and merge the project.

[Optional] We could show the error in the calculator display instead of in an alert.

**Pull request merged**